drivers/gles3/rasterizer_scene_gles3.h

```diff
@@ -207,6 +207,9 @@ class RasterizerSceneGLES3 : public RendererSceneRender {
 		uint32_t bake_mode;
 		float shadow_opacity;
 		float specular;
+
+		float pad[3];
+		uint32_t mask;
```

Comment:
To avoid increasing the size of the struct, `enabled` and `bake_mode` should be combined into a single uint32_t